lib/web/fetch/webidl.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const { types } = require('node:util')
+const { types, inspect } = require('node:util')
```

Comment:
I dont want to be nagging, but inspect is imho slow. will this have perf impact?

Reply:
it's only used for error cases